test/unit/sync/installation.test.ts

```diff
@@ -1,14 +1,23 @@
-import { isRetryableWithSmallerRequest, maybeScheduleNextTask, processInstallation } from "../../../src/sync/installation";
+import {
+	handleBackfillError,
+	isRetryableWithSmallerRequest, maybeScheduleNextTask,
+	processInstallation,
+} from "../../../src/sync/installation";
 
-import { DeduplicatorResult } from "../../../src/sync/deduplicator";
+import * as installation from "../../../src/sync/installation";
```

Comment:
Seems that star import works well alongside with named imports. Not sure if it is the best practice, however seems that the rest of the code will be more clear if I don't have to specify "installation." prefix before each imported entity